docs/topic_guides/splink_fundamentals/backends/backends.md

```diff
@@ -148,4 +148,4 @@ However, there are a couple of points to note:
 ```py
 SQLiteAPI(register_udfs=False)
 ```
-* As these functions are implemented in python they will be considerably slower than any native-SQL comparisons. If you find that your model-training or predictions are taking a large time to run, you may wish to consider instead switching to DuckDB (or some other backend).
\ No newline at end of file
+* As these functions are implemented in python they will be considerably slower than any native-SQL comparisons. If you find that your model-training or predictions are taking a large time to run, you may wish to consider instead switching to DuckDB (or some other backend).
```

Comment:
Not sure what changed here, but Github's GUI editor changed something in this line on save (perhaps whitespace?).